### DISCUSSION TITLE
UTM doesn't see any bridge in macOS.

### DISCUSSION THREAD

**Discussion by esrsound** [utmapp/UTM#6121] [UTM Help (macOS)]
**Describe the issue**  
UTM doesn't see any virtual bridge from macOS. I created a bridge in macOS network settings, but UTM doesn't see it and only sees the en0 interface. 

**Configuration**  
* UTM Version: 4.4.5
* macOS Version: 13.6
* M1 Pro

**Comment by osy** [comment]
You need to select bridged mode and that will create a bridge for you. You cannot create a bridge elsewhere and connect to it.
